azure-resources/Network/azureFirewalls/recommendations.yaml

```diff
@@ -97,7 +97,7 @@
     - name: Azure Well-Architected Framework review - Azure Firewall
       url: "https://learn.microsoft.com/en-us/azure/well-architected/service-guides/azure-firewall#recommendations"
 
-- description: Monitor AZFW Latency Probes metric
+- description: Monitor "AZFW Latency Probe" metrics
```

Comment:
Instead of putting quotes around the metric name, perhaps **bolding** it would be better. I'm not sure how this impacts our wara runner script(s), but something @Claudio-Merola can answer. I would suggest the following if it doesn't impact the WARA runner script.

```suggestion
- description: Monitor **AZFW Latency Probe** metric
```

Reply:
No, either the quotes or the bolding will affect the script.

Reply:
Thank you @Claudio-Merola for verifying that using quotes or bolding the text in the description has no impact on the WARA scripts.
@tksh164, I'm good with either bold or quotes. I do ask that you remove the 's' in metrics to only be metric. Otherwise, it LGTM.
Thx

Reply:
Thank you @DaFitRobsta for your review comments. I tested quickly with local file tweak then the Markdown notation affected the output files and the hugo generated document. See below. So, I will choose quotes.

![image](https://github.com/Azure/Azure-Proactive-Resiliency-Library-v2/assets/1618784/df056880-76b9-4806-bfd8-fb566448ed50)

![image](https://github.com/Azure/Azure-Proactive-Resiliency-Library-v2/assets/1618784/8da81c15-1f06-48fb-ba31-fb40403070c1)

![image](https://github.com/Azure/Azure-Proactive-Resiliency-Library-v2/assets/1618784/c38b4331-ff7b-4676-a3cd-468ee7581421)